<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreciseTreatment</class>
 <widget class="QWidget" name="PreciseTreatment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>精准治疗</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户设置</string>
   </property>
  </widget>
  <widget class="QComboBox" name="userSetComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>新建</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>编辑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>保存到U盘</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>读取U盘数据</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>肤色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skinColorBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/icons/icons/021.png);
background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>文字1</string>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>文字2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/icons/icons/delete30.png);
background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>test</string>
   </property>
  </widget>
  <widget class="QWidget" name="gearsGroup" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <widget class="QwtDial" name="gears">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lowerBound">
     <double>1.000000000000000</double>
    </property>
    <property name="upperBound">
     <double>15.000000000000000</double>
    </property>
    <property name="scaleMaxMajor">
     <number>15</number>
    </property>
    <property name="scaleMaxMinor">
     <number>1</number>
    </property>
    <property name="scaleStepSize">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
    <property name="totalSteps">
     <UInt>14</UInt>
    </property>
    <property name="pageSteps">
     <UInt>1</UInt>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="frameShadow">
     <enum>QwtDial::Sunken</enum>
    </property>
    <property name="mode">
     <enum>QwtDial::RotateNeedle</enum>
    </property>
    <property name="origin">
     <double>180.000000000000000</double>
    </property>
    <property name="maxScaleArc">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stateBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>准备</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeSurplus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>3</minute>
      <second>10</second>
     </time>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="skinColorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>III</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
